docs/serving/load-balancing/target-burst-capacity.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configuring target burst capacity"
+linkTitle: "Configuring target burst capacity"
+weight: 50
+type: "docs"
+aliases:
+    - /docs/serving/autoscaling/target-burst-capacity
+---
+
+_Target burst capacity_ determines the size of traffic burst a Knative application can handle without buffering.
+If a traffic burst is too large for an application to handle without buffering, the activator will be placed in the request path to protect the revision and optimize request load balancing.
+The autoscaler can then quickly spin up additional pods for capacity as required, and throttle how quickly requests are sent to pods.
+
+You can configure target burst capacity using the `autoscaling.knative.dev/targetBurstCapacity` annotation key in `config-autoscaler` ConfigMap, as shown in the following example:
+
+* **Global key:** No global key.
```

Comment:
there is a global value
`target-burst-capacity: "200"` in the config-autoscaling configmap.